src/mam4xx/gas_chem_mechanism.hpp

```diff
@@ -30,6 +30,19 @@ constexpr Real adv_mass[gas_pcnst] = {
     250092.672000, 1.007400,   12.011000,     12.011000, 250092.672000,
     1.007400};
 
+KOKKOS_INLINE_FUNCTION
```

Comment:
Is there a way to combine this and the above array? If it involves too many changes, let's do that latter.

Reply:
It will require more changes in other parts of the code. I will create an issue, and we can address it in a future PR.

Reply:
see issue: https://github.com/eagles-project/mam4xx/issues/364